src/lib/holocene/modal.svelte

```diff
@@ -44,10 +54,12 @@
 
   const handleCancel = () => {
     modalElement.close();
+    error = '';
     dispatch('cancelModal');
   };
 
   const confirmModal = () => {
+    error = '';
```

Comment:
It seems like this might be redundant in cases where we call `modal.close()` when handling confirm. Maybe we can handle the close here instead?

Reply:
@laurakwhit that's how the modal worked previous to this change, but now we want to keep the modal open if an error occurs in the request that's triggered when the modal is confirmed/submitted, so the modal can't really close itself unless it has knowledge of whether the request succeeds.

Reply:
I guess I was thinking we could somehow have a try/catch here as well, but it's definitely fine as is! Just trying to think through if there is a cleaner way to handle this...